lib/Doctrine/ORM/Mapping/Driver/AttributeDriver.php

```diff
@@ -645,19 +645,26 @@ private function getMethodCallbacks(ReflectionMethod $method): array
      *                   nullable: bool,
      *                   onDelete: mixed,
      *                   columnDefinition: string|null,
-     *                   referencedColumnName: string
+     *                   referencedColumnName: string,
+     *                   options?: mixed[]
```

Comment:
Let's use something more precise, if we have an exhaustive list? `array{default: mixed, unsigned: bool}` etc.

Reply:
This is the same as https://github.com/doctrine/orm/blob/7e49c703202fff2e3a9c753a496a2ac5bc342931/lib/Doctrine/ORM/Mapping/Column.php#L56-L57. I think it would be weird to start typing it here, and not do that for Column.

Reply:
I agree, that can be dealt with separately

Reply:
Okay, but it should be…

```suggestion
     *                   options?: array<string, mixed>
```

… then.

Reply:
@derrabus I changed it everywhere to your suggestion.